### PR TITLE
Reflow the login form to be inline (Close #71)

### LINKED ISSUE #71: Describe Caseflow on login screen
The large text at the VACOLS login prompt went relatively unnoticed by most participants. This text could include vital informaiton such as context about Caseflow and what it does.    

Suggestion: Consider making the text smaller and providing more explanation or context for what the application does here. Consider moving it to the left or right of the VACOLS username and password boxes.

### PR DESCRIPTION
**Pull request by paultag**
Bring it up side-by-side on medium and large screens, and move it back
to how it was when we're on a small size screen.

### CODE REVIEW

**Comment by paultag** [comment]
Sigh. Rebaseing. Screenshots next.

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -47,3 +47,20 @@ label {
     color: #000000;
   }
 }
+
+
+/* Login view */
+#login-login-button {
+    padding-top: 1em;
+}
```
Comment (alex) [inline app/assets/stylesheets/application.css.scss]: nit: our css uses 2space indents

File: app/views/web/login.html.erb [hunk 2]
```diff
@@ -15,40 +10,47 @@
...
-  <div class="row">
-    <div class="col col-md-12">
-      <label for="<%= username_input_id %>">Username</label>
-    </div>
-  </div>
-
   <div class="row">
     <div class="col col-md-6">
-      <input name="username" type="text" id="<%= username_input_id %>" autofocus="autofocus" />
+        <h1>Welcome to Caseflow!</h1>
```
Comment (alex) [inline app/views/web/login.html.erb]: same with our html

**Comment by alex** [comment]
can you add a screenshot for reference?

**Comment by paultag** [comment]
screenshot from 2015-10-29 11 42 29
screenshot from 2015-10-29 11 42 40

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (paultag) [inline app/assets/stylesheets/application.css.scss]: Gah, tended to.

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -47,3 +47,20 @@ label {
...
 }
+
+
+/* Login view */
+#login-login-button {
+  padding-top: 1em;
+}
+
+/* Grid sizes can be found on http://getbootstrap.com/css/ */
+@media (min-width: 970px) {
```
Comment (alex) [inline app/assets/stylesheets/application.css.scss]: AFAIK, media queries don't work in IE. What happens there?

**Comment by KHarshawat** [comment]
@paultag Rather than using the full 12 column width for these (6 for the left and 6 for the right) can we make them smaller. Say 4 (w/ col-offset-2) and 4? Or maybe 3+3 ... dunno what'll look like.

**Comment by KHarshawat** [comment]
(blue banner can stay the same width obvs)

**Comment by paultag** [comment]
@KHarshawat do you mean something like 
image or do you mean adding "padding" around the columns?

**Comment by paultag** [comment]
@KHarshawat - now set to image

From a 12 column set to `1, 5, 5, 1`

**Comment by alex** [comment]
@paultag needs a merge/rebase master

**Comment by paultag** [comment]
@alex rebased; r4r

**Comment by paultag** [comment]
and my @alex I mean @KHarshawat

**Comment by paultag** [comment]
image

@KHarshawat Updated!

**Comment by paultag** [comment]
:confetti_ball:

**Pull request merged**